### PR TITLE
Added entries for ZWyvern and Nardovern

### PR DESCRIPTION
**Pull request by madkat**
Added entries for both the [ZWyvern](https://morghus.gumroad.com/l/zwyvern) and [Nardovern](https://morghus.gumroad.com/l/nardovern) by Morghus. These are publicly available edits of the [ZDragon](https://zephyxus.gumroad.com/l/zdragon) and [Nardoragon](https://nardoiri.gumroad.com/l/Nardoragon) that integrate wings into the arms.

### CODE REVIEW

**Comment by jennytoo** [comment]
Do these share the armatures with the ZDragon and Nardoragon, or do they have distinct armatures?

**Comment by madkat** [comment]
They are distinct armatures with unique hash values. The fingers are the bones in the wings on the wyvern variants.

**Review by Happyrobot33 — CHANGES_REQUESTED** [review]
Just need to make sure these are v2 hashes, other than that looks good

File: Packages/com.mattshark.openflight/Runtime/data.json [hunk 1]
```diff
@@ -591,6 +591,26 @@
         "Weight": 1,
         "WingtipOffset": 10.5
       }
+    },
+    "ZWyvern": {
+      "ZWyvern": {
+        "Name": "ZWyvern",
+        "Creator": "Morghus",
+        "Introducer": "Literally Marty",
+        "Hash": ["1257052648"],
```
[the remark this answers is not in the document]
Comment (madkat) [reply Packages/com.mattshark.openflight/Runtime/data.json:600]: My mistake! I'll update to 1.2.7 later today and get the v2 hashes.

[the remark this answers is not in the document]
Comment (madkat) [reply Packages/com.mattshark.openflight/Runtime/data.json]: Replaced v1 hashes with v2 hashes.

File: Packages/com.mattshark.openflight/Runtime/data.json [hunk 2]
```diff
@@ -591,6 +591,26 @@
...
+        "Weight": 1,
+        "WingtipOffset": 10.2
+      }
+    },
+    "Nardovern": {
+      "Nardovern": {
+        "Name": "Nardovern",
+        "Creator": "Morghus",
+        "Introducer": "Literally Marty",
+        "Hash": ["1318433413"],
```
[the remark this answers is not in the document]
Comment (madkat) [reply Packages/com.mattshark.openflight/Runtime/data.json]: Replaced v1 hashes with v2 hashes.

**Review by Happyrobot33 — APPROVED** [review]

**Pull request merged**